graph/pull_v2_test.go

```diff
@@ -167,28 +167,4 @@ func TestValidateManifest(t *testing.T) {
 	if err == nil || !strings.HasPrefix(err.Error(), "image verification failed for digest") {
 		t.Fatal("expected validateManifest to fail with digest error")
 	}
-
-	// Manifest with no signature
-
-	expectedWholeFileDigest := "7ec3615a120efcdfc270e9c7ea4183330775a3e52a09e2efb194b9a7c18e5ff7"
```

Comment:
In the current world, it might be a bug that one cannot unmarshal into an SignedManifest.

And yes, this is completely against what I said above. Let's make sure that we are at least testing `validateManifest`.